docs/ui/components/Snippet/blocks/Terminal.tsx

```diff
@@ -77,20 +80,22 @@ function cmdMapper(line: string, index: number) {
     return (
       <div key={key}>
         <CODE
-          css={[
-            codeStyle,
-            unselectableStyle,
-            { display: 'inline', color: darkTheme.text.secondary },
-          ]}>
+          css={[{ display: 'inline', color: darkTheme.text.secondary }]}
+          className="whitespace-pre inline-block !bg-[transparent] !border-none text-default !leading-snug select-none">
```

Comment:
```suggestion
          css={[{ color: darkTheme.text.secondary }]}
          className="inline whitespace-pre inline-block !bg-[transparent] !border-none text-default !leading-snug select-none">
```

One more style converted.

Reply:
That's true, only one display property should be added, then whole `inline` style can be removed.